ansible/roles/ocp4-workload-integreatly/files/threescale/create-tenant.yml

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
Looking at other roles, they seem to keep all the tasks under one of the workloads. I don't know if it is common practice to do that, or break out tasks into it own file like this.

Reply:
I don't think it matters, these workflows will get very busy when we add in fuse tenants